### PR TITLE
Set autoscale params for node reducement

### PR DESCRIPTION
**Pull request by Balazs23**
**Description**

Changes proposed in this pull request:

- change autoscaler params to reduce nodes

**Related issue(s)**

### CODE REVIEW

**Review by jasiu001 — DISMISSED** [review]

File: components/kyma-environment-broker/internal/provider/azure_provider.go [hunk 1]
```diff
@@ -39,10 +39,10 @@ func (p *AzureInput) Defaults() *gqlschema.ClusterConfigInput {
 			Region:         DefaultAzureRegion,
```
Comment (kjaksik) [inline components/kyma-environment-broker/internal/provider/azure_provider.go:39]: Please remember to bump the KEB image

**Review by kjaksik — APPROVED** [review]

**Pull request merged**